### DISCUSSION TITLE
Change default parameter values

### DISCUSSION THREAD

**Discussion by brandon-gradient** [Opinionated-Blueprints/10-Books-Full-of-Rails#60] [Ideas]
This is mainly in reference to https://github.com/Opinionated-Blueprints/10-Books-Full-of-Rails/issues/54#issuecomment-2486902556 where you can't have the same default value in a parameterized BP since they will be merged.

![pic from issue](https://github.com/user-attachments/assets/5261b4d8-aa33-4683-8005-4e4103c2fee6)

I know this is probably a nit-pick but If I was going to pick on to be `4` as the default it would be the chests per wagon since your book almost exclusively uses this number of chests per wagon in all the areas I can think of off the top of my head, where as the number of cargo wagons a given stop is meant to handle is actually very variable. The book has 1, 2, and 4 as built in options for number of wagons. 
I would suggest if you're ever inclined, to switch them. This then means cargo wagons default could be `2` (pic has 2 there but the user who took it had already modified that value, default is 4) since it seems 1 is also already taken. Or `-2` if you want to be explicit like you were attempting to by having chests per wagon be -2

**Comment by Bocian-1** [comment]
Yeah, I'll do that. I didn't expect so many people to see a negative number and not realise something has to be done with it :p

**Comment by Bocian-1** [comment]
Implemented in [v2.19.0](https://github.com/Opinionated-Blueprints/10-Books-Full-of-Rails/releases/tag/v2.19.0)
